builders/config.json.example

```diff
@@ -1,3 +1,19 @@
+# It's critical to correctly configure the credentials
```

Comment:
I don't think it is right to write comments in the json file because the format doesn't support it. Maybe we can create a README.md file in the same folder?

Reply:
I fixed it.